docs/tutorial/inline.md

```diff
@@ -472,8 +437,42 @@ There it should be precise enough for the recursion to type but will get more pr
 > This implies that the body plays a part in the binary and source compatibility of this interface.
 
 
+## Compiletime Operations
+
+We also provide some operations that evaluate at compiletime.
```

Comment:
@nicolasstucki  Grouping inline matches under "Compiletime Operations" suggests that it should be moved to another section? Should we rename "Scala Compiletime Operations" to "Compiletime Operations" and also discuss `inline match` there? 

Or asked the other way around: why don't you like to group it with `inline if`?

Reply:
I'm not sure what to do with `inline match`. It does not belong with `inline if` because it would be misleading as `if` and `inline if` have the same semantics but `match` and `inline match` do not. Inline match behaves more like the methods in `compiletime`. It seems like `inline match` is closest to `summonFrom`. Maybe it should be moved to the `Compiletime Operations` document.